packages/flutter_tools/lib/src/flutter_manifest.dart

```diff
@@ -165,13 +165,10 @@ class FlutterManifest {
   ///   licenses:
   ///     - assets/foo_license.txt
   /// ```
-  List<String> get additionalLicenses {
-    final Object? licenses = _flutterDescriptor['licenses'];
-    if (licenses is YamlList) {
-      return licenses.map((Object? element) => element.toString()).toList();
-    }
-    return <String>[];
-  }
+  List<String> get additionalLicenses => <String>[
```

Comment:
This getter should still use curly braces. `=>` is typically reserved for single line returns.